pkg/opensearch/client/client.go

```diff
@@ -622,9 +622,6 @@ func (c *baseClientImpl) ExecutePPLQuery(ctx context.Context, r *PPLRequest) (*P
 			},
 		}
 	}
-	if err != nil {
```

Comment:
this was just a linter thing that was annoying me. err can never be non-nil at this point